apps/rush-lib/src/cli/RushCommandLineParser.ts

```diff
@@ -117,30 +117,31 @@ export class RushCommandLineParser extends CommandLineParser {
   }
 
   private _populateScriptActions(): void {
-    if (!this.rushConfiguration) {
-      return;
-    }
+    let commandLineConfiguration: CommandLineConfiguration | undefined = undefined;
 
-    const commandLineConfigFile: string = path.join(
-      this.rushConfiguration.commonRushConfigFolder, RushConstants.commandLineFilename
-    );
-    const commandLineConfiguration: CommandLineConfiguration
-      = CommandLineConfiguration.loadFromFileOrDefault(commandLineConfigFile);
+    // If there is not a rush.json file, we still want "build" and "rebuild" to appear in the
+    // command-line help
+    if (this.rushConfiguration) {
+      const commandLineConfigFile: string = path.join(
+        this.rushConfiguration.commonRushConfigFolder, RushConstants.commandLineFilename
+      );
 
-    const documentationForBuild: string = 'The Rush build command assumes that the package.json file for each'
-      + ' project contains a "scripts" entry for "npm run build".  It invokes'
-      + ' this commands to build each project.  Projects are built in parallel where'
-      + ' possible, but always respecting the dependency graph for locally linked projects.'
-      + ' The number of simultaneous processes will be based on the number of machine cores'
-      + ' unless overridden by the --parallelism flag.';
+      commandLineConfiguration = CommandLineConfiguration.loadFromFileOrDefault(commandLineConfigFile);
+    }
 
     // always create a build and a rebuild command
     this.addAction(new BulkScriptAction({
       actionName: 'build',
       summary: '(EXPERIMENTAL) Build all projects that haven\'t been built, or have changed since they were last '
         + 'built.',
-      documentation: documentationForBuild,
-
+      documentation: 'This command is similar to "rush rebuild", except that "rush build" performs'
+        + ' an incremental build. In other words, it only builds projects whose source files have'
+        + ' changed since the last successful build. The analysis requires a Git working tree, and'
+        + ' only considers source files that are tracked by Git and whose path is under the project folder.'
```

Comment:
>only considers source files that are tracked by Git [](start = 12, length = 51)

More precisely "only consider files that aren't ignored by Git" since it does take into account untracked files.